pom.xml

```diff
@@ -588,6 +558,19 @@
         </pluginManagement>
         <plugins>
             <plugin>
+                <groupId>org.codehaus.gmavenplus</groupId>
+                <artifactId>gmavenplus-plugin</artifactId>
+                <version>1.4</version>
```

Comment:
why the different version here than above?  Shouldn't this be a parameter in the pom?